bigchaindb/common/transaction.py

```diff
@@ -118,7 +118,7 @@ def from_dict(cls, data):
             fulfillment = Fulfillment.from_uri(data['fulfillment'])
         except ValueError:
             # TODO FOR CC: Throw an `InvalidSignature` error in this case.
-            raise InvalidSignature("Fulfillment URI couldn't been parsed")
+            raise InvalidSignature('Fulfillment URI couldn\'t been parsed')
```

Comment:
Sentences with internal single quotes, like this one, are one situation where I think double quotes can make the line more readable, but I'm not going to make a big fuss over it. Maybe we can just standardize on a single-quotes-always rule. If so, you should update this section of our Python style guide:   https://github.com/bigchaindb/bigchaindb/blob/master/PYTHON_STYLE_GUIDE.md#single-or-double-quotes